backend/notifications/permissions.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from rest_framework.permissions import BasePermission
+from rest_framework.authentication import BasicAuthentication
+from rest_framework.exceptions import AuthenticationFailed
+
+
+class SNSAuthentication(BasicAuthentication):
```

Comment:
turns out basic auth has been broken on production (probably due to the api gateway to lambda URL migration?) so this doesn't work.

aws renamed auth header to `x-amzn-remapped`. find a solution